### PR TITLE
Add support for getting groups that a user belongs to

### PR DESCRIPTION
**Pull request by lboynton**
This adds support for getting the groups that a user belongs to.

https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_ref_usersgroups.htm#get_groups_for_a_user

### CODE REVIEW

File: tableauserverclient/server/endpoint/users_endpoint.py [hunk 1]
```diff
@@ -96,3 +96,23 @@ def _get_wbs_for_user(self, user_item, req_options=None):
...
         self.parent_srv.favorites.get(user_item)
+
+    # Get groups for user
+    @api(version="3.7")
+    def populate_groups(self, user_item, req_options=None):
+        if not user_item.id:
+            error = "User item missing ID."
+            raise MissingRequiredFieldError(error)
+
+        def wb_pager():
```
Comment (t8y8) [inline tableauserverclient/server/endpoint/users_endpoint.py:107]: May as well give this a more descriptive name, maybe `groups_for_user_pager`

**Review by t8y8 — DISMISSED** [review]
🚀 

Just one small request

**Review by shinchris — APPROVED** [review]
Thanks for adding this!
Could you change the base branch to `development` rather than `master`?

**Comment by lboynton** [comment]
> Thanks for adding this!
> Could you change the base branch to `development` rather than `master`?

Ah yes sorry, have rebased on the development branch!

**Comment by shinchris** [comment]
One more thing before we merge - have you had a chance to read and sign our [CLA](https://tableau.github.io/contributing.html)?

**Review by t8y8 — APPROVED** [review]
🚀

**Comment by lboynton** [comment]
> One more thing before we merge - have you had a chance to read and sign our [CLA](https://tableau.github.io/contributing.html)?

Have just sent the signed PDF :)

**Comment by bcantoni** [comment]
Confirming I've received a signed CLA from @lboynton and we are good to go.

**Pull request merged**